tests/queries/0_stateless/02252_jit_profile_events.reference

```diff
@@ -1,4 +1,4 @@
 0
 1
 0	1	2
-1
+3
```

Comment:
This test should be adjusted to it keeps testing the same thing. Maybe replace `sum` with `avg`

Reply:
Done.